appendix-basic65.tex

```diff
@@ -3457,9 +3463,19 @@ \subsection{EDMA}
                  target [{\bf,} sub{\bf,} mod]
 \item [Usage:]
    {\bf EDMA} ("Extended Direct Memory Access") is the fastest method
-   to manipulate memory areas using the DMA controller.
+   to manipulate memory areas using the DMA controller. Please refer to
+\ifdefined\printmanual
+the {\bf MEGA65 Book}
+\else
+\bookvref{cha:dmagic}
+\fi
+   for the powerful possibilities with EDMA.
 
    {\bf command} 0: copy, 1: mix, 2: swap, 3: fill.
+   
+   Because this two bits of the command share the same register with other
+   bits you can for example use bit 5 to reverse loop operation. This is
+   also helpful for in-place memory displacing. Please see the example below.
```

Comment:
If you set both bits the effect is reversed instantly. It is then only filling in reverse order but also end first. In other words LGB's screenshot would no longer be accurate.

![basic-example-edma](https://user-images.githubusercontent.com/67829169/210105369-62d269d9-c088-4312-82e1-ef1643889949.png)

> &hellip;helpful for in-place memory displacing&hellip;

Yeah, some stupid wording of mine for what Paul was suggesting in Discord. What's actually meant is if you want to move in between overlapping source and destination memory regions. Like caterpillaring strings around the character screen. Do you remember Snake on Nokia phones? ;) I'm now merging:

```
   Because this two bits of the command share the same register with other
   bits you can for example use bit 5 to reverse loop operation. This is
   also working in overlapping memory regions for source and target. Please
   see the example below.
```

Okay?